master/internal/api_experiment.go

```diff
@@ -75,28 +75,56 @@ func (a *apiServer) getExperiment(experimentID int) (*experimentv1.Experiment, e
 }
 
 func (a *apiServer) GetSearcherEvents(_ context.Context, req *apiv1.GetSearcherEventsRequest) (
-	*apiv1.GetSearcherEventsResponse, error) {
+	resp *apiv1.GetSearcherEventsResponse, err error) {
 	exp, err := a.getExperiment(int(req.ExperimentId))
 	if err != nil {
-		return nil, errors.Wrap(err, "fetching experiment from db")
+		return nil, err
+	}
+	if exp.State == experimentv1.State_STATE_COMPLETED {
+		eventCount := 1000 // TODO get event count from experiment
```

Comment:
This is a problem we have to address, either in this PR or later. Since the experiment is gone we don't have access to the searcher or the search method or the queue. The only thing I can think of is save the number of events in the experiment in the DB. Suggestions are welcome.

Reply:
I can address it in this ticket: https://determinedai.atlassian.net/browse/DET-7717.

Reply:
Right so I was thinking of returning a list like this: [ExperimentInactive] then it'll look similar to the events list in the SearcherQueue.